www/lang/code.js

```diff
@@ -105,4 +106,4 @@ Code.initLanguage = function() {
 document.write('<script src="lang/msg_' + Code.LANG + '.js"></script>\n');
 document.write('<script src="lang/Blockly_' + Code.LANG + '.js"></script>\n');
 document.write('<script src="lang/microbit_' + Code.LANG + '.js"></script>\n');
-document.write('<script src="lang/Arduino_' + Code.LANG + '.js"></script>\n');
\ No newline at end of file
+document.write('<script src="lang/Arduino_' + Code.LANG + '.js"></script>\n');
```

Comment:
@Chacha75 what was this change?

Reply:
I think only the 'carriage return' and 'line feed' characters changed due to the editor. (It was the gitHub editor) I haven't changed anything in this line.